### PR TITLE
Add new signing keys with notification

### PR DESCRIPTION
**Pull request by sanderson**
This PR updates all the signing key file names, checksums, and signing key references through the documentation. It also adds a notification to any installation pages that were affected:

image

- [x] Rebased/mergeable

### CODE REVIEW

**Review by gunnaraasen — APPROVED** [review]
LGTM

File: content/enterprise_influxdb/v1.10/introduction/installation/data_node_installation.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ For added security, follow these steps to verify the signature of your InfluxDB
 
 1. Download and import InfluxData's public key:
    ```
-   curl -s https://repos.influxdata.com/influxdb.key | gpg --import
+   curl -s https://repos.influxdata.com/influxdata-archive.key | gpg --import
```
Comment (jdstrand) [inline content/enterprise_influxdb/v1.10/introduction/installation/data_node_installation.md:130]: Here and elsewhere, for maximum compatibility, this should be `influxdata-archive_compat.key`, not `influxdata-archive.key`. See the FAQ of the blog post for details (the updates for the `-oss` filename change are being rolled out presently).

(Also, thank you for proactively adjusted to remove `-oss` from the filename).

File: content/influxdb/v1.8/introduction/install.md [hunk 2]
```diff
@@ -68,18 +68,18 @@ For Ubuntu/Debian users, add the InfluxData repository with the following comman
 {{% /code-tabs %}}
 {{% code-tab-content %}}
 ```sh
-# influxdb.key GPG Fingerprint: 05CE15085FC09D18E99EFB22684A14CF2582E0C5
-wget -q https://repos.influxdata.com/influxdb.key
-echo '23a1c8836f0afc5ed24e0486339d7cc8f6790b83886c4c96995b88a061c5bb5d influxdb.key' | sha256sum -c && cat influxdb.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdb.gpg > /dev/null
+# influxdata-archive.key GPG Fingerprint: 9D539D90D3328DC7D6C8D3B9D8FF8E1F7DF8B07E
+wget -q https://repos.influxdata.com/influxdata-archive.key
+echo '393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c influxdata-archive.key' | sha256sum -c && cat influxdata-archive.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdb.gpg > /dev/null
```
Comment (jdstrand) [inline content/influxdb/v1.8/introduction/install.md:73]: As mentioned, here and elsewhere, please use `influxdata-archive_compat.key`. GPG fingerprint and sha256 remain the same as what you have listed.

**Review by jdstrand — CHANGES_REQUESTED** [review]
Thanks for the PR! Please see inline.

File: content/influxdb/v1.8/introduction/install.md [hunk 3]
```diff
@@ -68,18 +68,18 @@ For Ubuntu/Debian users, add the InfluxData repository with the following comman
 {{% /code-tabs %}}
 {{% code-tab-content %}}
 ```sh
-# influxdb.key GPG Fingerprint: 05CE15085FC09D18E99EFB22684A14CF2582E0C5
-wget -q https://repos.influxdata.com/influxdb.key
-echo '23a1c8836f0afc5ed24e0486339d7cc8f6790b83886c4c96995b88a061c5bb5d influxdb.key' | sha256sum -c && cat influxdb.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdb.gpg > /dev/null
+# influxdata-archive_compat.key GPG Fingerprint: 9D539D90D3328DC7D6C8D3B9D8FF8E1F7DF8B07E
+wget -q https://repos.influxdata.com/influxdata-archive_compat.key
+echo '393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c influxdata-archive_compat.key' | sha256sum -c && cat influxdata-archive_compat.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdb.gpg > /dev/null
 echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdb.gpg] https://repos.influxdata.com/debian stable main' | sudo tee /etc/apt/sources.list.d/influxdata.list
```
Comment (jdstrand) [inline content/influxdb/v1.8/introduction/install.md:74]: Lines like this (here and elsewhere) need changing to `echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main' | sudo tee /etc/apt/sources.list.d/influxdata.list`

**Review by jdstrand — CHANGES_REQUESTED** [review]
Thanks for the updates! Couple more changes inline to ensure we match what is in the blog and what is in the instructions for the downloads page.

**Review by jdstrand — CHANGES_REQUESTED** [review]
Ok, I *think* this is the last batch. Things are looking really good. I downloaded the branch locally and noticed that `influxdb2.key` is being referenced in a few places (these binaries will also be re-signed):

```
$ grep -r 'repos\.influxdata\.com' ./*|grep -v influxdata-archive_compat.key|grep key
./content/influxdb/v2.0/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.0/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.4/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.4/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.3/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.3/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.2/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.2/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.6/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.6/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.5/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.5/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.1/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
./content/influxdb/v2.1/install.md:    curl -s https://repos.influxdata.com/influxdb2.key | gpg --import -
```
(these should be updated to use `influxdata-archive_compat.key` like the others)

```
$ grep -r 'apt/trusted' ./*|grep -v influxdata-archive_compat.gpg
./content/telegraf/v1.20/introduction/installation.md:wget -qO- https://repos.influxdata.com/influxdata-archive_compat.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
./content/telegraf/v1.20/introduction/installation.md:curl -s https://repos.influxdata.com/influxdata-archive_compat.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
./content/telegraf/v1.22/install.md:wget -qO- https://repos.influxdata.com/influxdata-archive_compat.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
./content/telegraf/v1.22/install.md:curl -s https://repos.influxdata.com/influxdata-archive_compat.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
./content/telegraf/v1.21/introduction/installation.md:wget -qO- https://repos.influxdata.com/influxdata-archive_compat.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
./content/telegraf/v1.21/introduction/installation.md:curl -s https://repos.influxdata.com/influxdata-archive_compat.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
```
(`influxdb.asc` should be updated to be `influxdata-archive_compat.gpg`

**Review by jdstrand — COMMENTED** [review]
Approving post-merge. Thanks for the changes.

**Pull request merged**